docs/rules/expect-expect.md

```diff
@@ -28,3 +26,51 @@ it('should work with callbacks/async', () => {
   somePromise().then(res => expect(res).toBe('passed'));
 });
 ```
+
+## Options
+
+```json
+{
+  "jest/expect-expect": [
+    "error",
+    {
+      "allow": ["expect"]
+    }
+  ]
+}
+```
+
+### `allow`
```

Comment:
could we call it `assertNames` or `assertFunctions` or something like that instead?